lib/thrift/generator/server.ex

```diff
@@ -0,0 +1,143 @@
+defmodule Thrift.Generator.Server do
+  @moduledoc false
+
+  defmodule Framed do
```

Comment:
I can also save indentation by putting this in its own file / directory. 
you and @jparise need to duke this out; it's not fun going back and forth on this. Personally, I think BinaryFramedServer is redundant given it's in the Server.Framed module.

Either we make a module called Servers and have BinaryFramed in that or we keep the present nesting; I don't care which, but having both is redundant.